doc: update changelog for version 5.5.0 with exactly-once (#113)
neo4j/docs-kafka-connector: modules/ROOT/pages/source/query.adoc

In case you generate complex data structures as part of your Cypher query, you can also use the `RAW_JSON_STRING` payload mode which will produce a raw JSON string representation of the data without any schema compatibility guarantees.
Although the generated message will look exactly the same as the `COMPACT` mode, it will be encoded as a `STRING` type.

> == Exactly Once Semantics

Reviewer: Can you make the use of "exactly-once semantics" consistent in style? No capitals (except if "Exactly" is the first word, like in this header), hyphen.

> The query strategy declares itself as *not* Exactly Once capable, and provides at-least-once delivery guarantees.

Reviewer 2: ```suggestion
The query strategy is *not* Exactly Once capable, and provides at-least-once delivery guarantees.
```
